<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Previewer</class>
 <widget class="QWidget" name="Previewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Previewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
   <item>
    <widget class="QWidget" name="viewer" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="fileOpGrp">
     <property name="title">
      <string>Operations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="openDirBut">
        <property name="text">
         <string>Open Directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="cpyDirBut">
        <property name="text">
         <string>Copy Directory Path</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="infoLbl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="openFileBut">
        <property name="text">
         <string>Open File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="cpyFileBut">
        <property name="text">
         <string>Copy File Path</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="selBut">
     <property name="text">
      <string>Select File to Preview</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
